src/o.js

```diff
@@ -48,85 +48,89 @@ o.prototype.setAttributes = function (attributes) {
     return this.attr(attributes);
 }
 
-o.prototype.attr = function(attrs) {
+o.prototype.attr = function (attrs) {
     if (Array.isArray(attrs)) {
         attrs.forEach(attr => this.element.setAttribute(attr.name, attr.val));
     } else {
-        Object.entries(attrs).forEach(([name,val]) => this.element.setAttribute(name, val));
+        Object.entries(attrs).forEach(([name, val]) => this.element.setAttribute(name, val));
     }
     return this;
 };
 
-o.prototype.class = function(classNames) {
+o.prototype.class = function (classNames) {
     if (Array.isArray(classNames)) {
         classNames.forEach(className => this.element.classList.add(className));
-    }else if (typeof classNames === 'string') {
+    } else if (typeof classNames === 'string') {
         this.element.className = classNames;
     }
     return this;
 };
 
-o.prototype.classList = function(classList) {
+o.prototype.classList = function (classList) {
     return this.class(classList)
 };
-o.prototype.className = function(className) {
+o.prototype.className = function (className) {
     return this.class(className)
 }
 
-o.prototype.id = function(id) {
+o.prototype.id = function (id) {
     this.element.setAttribute('id', id);
     return this;
 };
 
-o.prototype.add = function(children) {
-    if (this.element instanceof oFragment){
-        this.element.add(children);
-    }else {
-        children.forEach(child => {
-            try {
-                if(typeof child === 'boolean')
```

Comment:
ten if czy child to boolean też był przydatny. Jak robileś opcjonalny element w .add() to jak nie spełniło warunku wrzucało false i tu pomijało. Więc to też trzeba by dodać